CBot/src/CBot/CBotProgram.h

```diff
@@ -60,12 +64,17 @@ class CBotExternalCallList;
  * // Initialize the engine
  * CBotProgram::Init();
  *
+ * // Create context for public functions, classes, etc.
+ * auto context = CBotContext::Create();
```

Comment:
```suggestion
 * auto context = CBotContext::CreateGlobalContext();
```

CBotContext::Create() has  a non-optional argument